pkg/cmd/initialize/sigs_test.go

```diff
@@ -8,41 +8,133 @@ import (
 	"github.com/aquasecurity/tracee/pkg/events"
 	"github.com/aquasecurity/tracee/pkg/signatures/signature"
 	"github.com/aquasecurity/tracee/types/detect"
+	"github.com/aquasecurity/tracee/types/trace"
 )
 
 func Test_CreateEventsFromSigs(t *testing.T) {
 	tests := []struct {
 		startId        events.ID
 		signatures     []detect.Signature
-		expectedEvents []events.Event
+		expectedEvents []*events.Event
 	}{
 		{
 			startId: events.ID(6001),
 			signatures: []detect.Signature{
-				newFakeSignature("fake_event_0", []string{"hooked_syscalls"}),
+				newFakeSignature(
+					"fake_event_0",
+					[]string{
+						"hooked_syscalls",
+					},
+				),
 			},
-			expectedEvents: []events.Event{
-				events.NewEventDefinition("fake_event_0", []string{"signatures", "default"}, []events.ID{events.HookedSyscalls}),
+			expectedEvents: []*events.Event{
+				events.NewEvent(
```

Comment:
I agree and was expecting the definitions to be discussed and done. EventDefinition is good enough for me, I was just worried because we use event := NewEvtDefinition() in many places, its misleading there as well (to understand if its a definition or a Tracee.Event).

I'll try to change this to make it more clear.